docker/bootstrap/Dockerfile.common

```diff
@@ -1,35 +1,22 @@
-FROM golang:1.13-stretch
+FROM golang:1.13-buster
 
 # Install Vitess build dependencies
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     # TODO(mberlin): Group these to make it easier to understand which library actually requires them.
-    automake \
-    bison \
-    bzip2 \
+    ant \
     chromium \
     curl \
+    default-jdk \
+    etcd \
```

Comment:
Not sure if we need this at all. I've only backported it from #6129

Reply:
@harshit-gangal should be ok to change the jdk version from openjdk-8 to default-jdk?

Reply:
we need to keep it at jdk8 otherwise java code will not compile. Any reason for changing this?

Reply:
> we need to keep it at jdk8 otherwise java code will not compile. Any reason for changing this?

Unfortunately there isn't `openjdk-8-jdk` available for Buster.

In `stretch`,  `default-jdk` points to the Java runtime, or Java compatible development kit recommended for this architecture, which is `openjdk-8-jdk` for `amd64` (see [doc](https://packages.debian.org/stretch/default-jdk)).

In `buster`, `default-jdk` points to the Java runtime, or Java compatible development kit recommended for this architecture, which is `openjdk-11-jdk` for `amd64` (see [doc](https://packages.debian.org/buster/default-jdk)).